docs/resources/aws_launch_configuration.md

```diff
@@ -0,0 +1,159 @@
+---
+title: About the aws_launch_configuration Resource
+---
+
+# aws\_launch\_configuration
+
+Use the `aws_launch_configuration` InSpec audit resource to test properties of a single AWS Launch Configuration. 
+
+<br>
+
+## Availability
```

Comment:
You can delete the Availability, Installation and Version headers

Reply:
👍 done